slack/Api/SlackApiService.php

```diff
@@ -62,10 +62,10 @@ public function getSlackMemberByEmail(SlackCredentials $credentials, string $ema
         throw new SlackMemberNotFoundException();
     }
 
-    public function sendMessageToChannel(SlackCredentials $credentials, string $message, string $channel): void
+    public function sendMessageToChannel(SlackCredentials $credentials, string $message, string $channel, string $blocks = null, string $attachments = null): void
     {
         $client = $this->getAuthenticateClient($credentials);
-        $client->chatPostMessage(['channel' => $channel, 'text' => $message]);
+        $client->chatPostMessage(['channel' => $channel, 'text' => $message, 'attachments' => $attachments, 'blocks' => $blocks]);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏le mieux est d'éviter d'envoyer des data `null`

```suggestion
        $payload = ['channel' => $channel, 'text' => $message];
        if (isset($attachments)) {
            $payload['attachments'] = $attachments;
        }
        if (isset($blocks)) {
            $payload['blocks'] = $blocks;
        }
        
        $client->chatPostMessage($payload);
```